screens/Menu/NearbyInterestDetails.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { View, Text, StyleSheet } from "react-native";
+import { Icon, Button } from "native-base";
+
+
+
+function NearbyInterestDetails(props) {
+
+    const goBack = () => {
+        props.navigation.goBack();
+    }
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.menuButtonContainer}>
+                <Button transparent style={styles.menuButton} onPress={goBack}>
+                <Icon name="md-arrow-round-back" style={styles.icon} />
+                </Button>
+            </View>
+
+            <Text style={styles.mainLabel}>Point of Interests Details</Text>
+            
+            <Button transparent style={styles.routeButton} ><Text style={styles.viewRouteText}>View Route</Text></Button>
```

Comment:
We are going to have to add these points of interest in the data file for them to appear in the search bar. This is an issue that you can create.

Reply:
Yes, I understand your idea. However, I was under the impression that the detail page for the points of interest were for the __places__ that we would retrieve from the the Google Places API. With just the _points_of_interest screen, I thought it would be enough, is it required of us to have it in search bar as well?

Reply:
True, good idea!